### ISSUE TITLE
Body text content of the modal is identified as clickable by screen reader

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4597]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 2.4.3 Focus Order

**Test Environment:**
- Test URL: https://clarity.design/documentation/modals
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0
- Screen reader: NVDA 19.3.1

**Steps to reproduce:**
1. Open the test URL.
2. Move focus onto to the first Show Modal button present under Angular modals.
3. Activate the button by pressing the enter or space key and observe the spoken feedback for the body content and also perform tab navigation.

**Expected result:**
Body content should be announced appropriately and tab navigation should allow user to tab through the clickable element.

**Actual result:**
Body content of the dialog is identified as clickable by screen reader and it also get&#039;s focused while performing tab navigation.

**Proposed solution:**
Remove tab index from the body content **div** tag.

**Reporter Email:** gkhadke@vmware.com

**Comment by aognianova** [comment]
I'm working on VMware Site Recovery Manager product and our VPAT report performed by Level Access 3rd party vendor has identified the same problem:

The body/content of the modal is receiving focus which is not necessary as this is a non interactive element. 

Their recommendation: Simply remove tabindex="0" on the modal div container and modal body div to remove it from the focus order.

**Comment by hippee-lee** [comment]
Hi @aognianova.
The issues and recommendations identified by VPAT's are not always the right way to fix either the application or Clarity.
Please work with your internal accessibility lead to triage and investigate the issue. It it is something that needs to change, they will batch and report issues like this with other similar issues for modal once there is an agreed solution for resolution.

**Comment by gnomeontherun** [comment]
Hi there, this is an automated message.

In an effort to dedupe and track accessibility reports consistently, this issue is being closed and tracked internally with our Accessibility team. It will still be triaged and addressed as part of ongoing work.
